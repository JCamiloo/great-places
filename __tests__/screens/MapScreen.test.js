import "react-native";
import React from "react";
import MapScreen from "../../screens/MapScreen";
import renderer from "react-test-renderer";
import configureMockStore from "redux-mock-store";
import { Provider } from "react-redux";

const mockStore = configureMockStore();
const store = mockStore({});

describe("MapScreen tests", () => {
  test("MapScreen renders correctly", () => {
    const tree = renderer
      .create(<MapScreen navigation={{ getParam: jest.fn() }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
