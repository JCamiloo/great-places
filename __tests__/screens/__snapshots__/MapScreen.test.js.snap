// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapScreen component tests MapScreen component renders correctly 1`] = `
<AIRMap
  handlePanDrag={false}
  initialRegion={null}
  onChange={[Function]}
  onLayout={[Function]}
  onMapReady={[Function]}
  onMarkerPress={[Function]}
  onPress={[Function]}
  region={
    Object {
      "latitude": 37.78,
      "latitudeDelta": 0.0922,
      "longitude": -122.43,
      "longitudeDelta": 0.0421,
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;

exports[`MapScreen tests MapScreen renders correctly 1`] = `
<AIRMap
  handlePanDrag={false}
  initialRegion={null}
  onChange={[Function]}
  onLayout={[Function]}
  onMapReady={[Function]}
  onMarkerPress={[Function]}
  onPress={[Function]}
  region={
    Object {
      "latitude": 37.78,
      "latitudeDelta": 0.0922,
      "longitude": -122.43,
      "longitudeDelta": 0.0421,
    }
  }
  style={
    Object {
      "flex": 1,
    }
  }
/>
`;
